frontend/src/common/enums/ui/label-names.enum.ts

```diff
@@ -0,0 +1,10 @@
+enum LabelNames {
```

Comment:
it's good that you want to use enums <3, but this is where they are not needed
remove it